pkg/kudoctl/cmd/get/get.go

```diff
@@ -48,7 +48,7 @@ func Run(args []string, opts CmdOpts) error {
 	case Operators:
 		objs, err = opts.Client.ListOperators(opts.Namespace)
 	case OperatorVersions:
-		objs, err = opts.Client.ListOperatorVersions(opts.Namespace)
+		objs, err = opts.Client.ListOperatorVersionsAsRuntimeObject(opts.Namespace)
```

Comment:
```suggestion
		objs, err = opts.Client.ListOperatorVersions(opts.Namespace)
```

Reply:
all previous client calls `ListOperators` and `ListInstances` also return `[]runtime.Object` with a shorter name... we should be consistent.

Reply:
Good catch. Done ✅